modelchecking/ltsmin/src/main/java/net/automatalib/modelcheckers/ltsmin/AbstractLTSmin.java

```diff
@@ -177,6 +178,27 @@ public Function<String, I> getString2Input() {
             throw new ModelCheckingException(ioe);
         }
 
+        final File ltlFile;
+
+        try {
+            // create a file that will contain the LTL of the specification
+            ltlFile = File.createTempFile("formula", ".ltl");
+
+            try {
+                // write to the file
+                FileWriter writer = new FileWriter(ltlFile);
+                writer.write(formula);
+                writer.close();
```

Comment:
Could you maybe use a "try-with-resources" block, so that Java takes care of closing streams and possible exception handling?